traiders/backend/api/models/portfolio.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from .users import User
+from .parity import Parity
+
+
+class Portfolio(models.Model):
+    name = models.CharField(max_length=64, blank=False)
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, blank=False)
```

Comment:
Can you rename `owner` to `user`? We followed the same convention in other models too, it is clear that the user "owns" the portfolio.

Reply:
done :+1: